src/Servers/IIS/IIS/test/FunctionalTest.props

```diff
@@ -1,25 +1,24 @@
 <Project>
-
   <ItemGroup>
     <Content Include="..\Common.FunctionalTests\Infrastructure\*.config" CopyToOutputDirectory="PreserveNewest" />
   </ItemGroup>
 
-  <Target Name="BuildAssets" AfterTargets="Build" Condition="'$(ExcludeFromBuild)' != 'true'">
-    <MSBuild Projects="@(ProjectReference)" Targets="PublishTestsAssets" SkipNonexistentTargets="true" BuildInParallel="True">
+  <Target Name="CopyAssets" BeforeTargets="Publish" Condition=" '$(ExcludeFromBuild)' != true ">
```

Comment:
doing the two parts in separate targets worked when `Publish` always depended on `Build`

Reply:
Is there any noticeable change here in behavior that we need to be aware of or is this just implementation details?

Reply:
Nothing I could detect because the published files from the test assets aren't used in local tests.